etc/adminhtml/system.xml

```diff
@@ -15,6 +15,12 @@
                         <field id="enabled">1</field>
                     </depends>
                 </field>
+                <field id="data_culture" translate="label comment" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
I think this should be a `select` field with all possible "two-letter ISO language codes" (https://support.cookiebot.com/hc/en-us/articles/360003816053-Make-CMS-control-language).

Reply:
I agree that we should keep it as it is.